allennlp/commands/predict.py

```diff
@@ -46,4 +45,4 @@ def predict(args: argparse.Namespace) -> None:
         if args.output_file:
             output_file = stack.enter_context(args.output_file)  # type: ignore
 
-        run(predictor, input_file, output_file, args.print)
+        run(predictor, input_file, output_file, not args.silent)
```

Comment:
(I thought I already commented on this, but I guess not)

as discussed, we should terminate early if no file is specified and also --silent is chosen

Reply:
sure, although I don't feel this is necessary now that we have `--silent` instead of `--print`.